packages/web/app/api/gql/typeDefs/lib.ts

```diff
@@ -57,12 +57,20 @@ type RiskScoreDetails {
 }
 
 type TokenMeta {
+  # Legacy fields
   type: String
-  category: String
-  description: String
+  icon: String
+  # Core fields from yCMS
+  chainId: Int
```

Comment:
`chainId`, `address`, `name`, and `registry` are convenience fields used by cms. do not include these in the graphql typedef. this goes for all the typedefs, VaultMeta, TokenMeta, and StrategyMeta